deploy/configure-assets-after-deployment.ts

```diff
@@ -9,43 +10,86 @@ import {
   getRestrictedNORI,
 } from '@/utils/contracts';
 
+const CONFIRMATIONS = 5;
 export const deploy: DeployFunction = async (environment) => {
   const hre = environment as unknown as CustomHardHatRuntimeEnvironment;
   Logger.setLogLevel(Logger.levels.DEBUG);
   hre.trace(`configure-assets-after-deployment`);
+  // const provider = new ethers.providers.JsonRpcProvider(
```

Comment:
If you'd like to keep this commented code for posterity it'd be nice to see a comment above it like you did elsewhere in this PR.